src/workloads/filesystem/Flushing.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-perf"
+Description: >
+  Run large insertMany's to stress checkpointing, journal flush and dirty memory with {w:majority}.
+  The workload runs a single CrudActor targeting:
+    - 'test' database, configurable via the Database parameter.
+    - 'Collection0' collection, configurable via the Collection parameter.
+    - 10 threads, configurable via the Threads parameter.
+    - 5000 iterations, configurable via the Iterations parameter.
+    - 1500 documents, configurable via the Documents parameter.
+    - {w:majority} insert options, configurable via the Options parameter.
+  The default documents have a very simple / flat structure as follows:
+    - _id: ObjectId
+    - x: a ~1KB fixed string 
+    - id: 'User%07d' incrementing field.
+  There are no additional indexes (just _id).
+  The important ftdc metrics to look at are:
+    - ss opcounters insert
+    - ss average latency writes
+    - system memory dirty
+    - ss wt log log sync time duration
+    - ss wt transaction transaction checkpoint currently running
+
+Keywords:
+  - Flushing
+  - IO
+  - bulk insert
+  - stress
+
+GlobalConstants:
+  - &database test
+  - &collection Collection0
+  - &maxPhases 1
+  - &insertThreads 10
+  - &iterationsCount 5000
+  - &defaultOptions
+    WriteConcern:
+      Level: majority
+  - &defaultDocument  # The size of each document is about 1kb.
+    x: {^PreprocessorFormatString: {format: "x%*s", withArgs: [1000, "x" ]}}
```

Comment:
Could you add a comment above this line describing the generated string. It's just non-obvious to me, but I think I can figure it out from context.

Reply:
will do.